# Data and scripts for "Adaptive cellular evolution in the intestines of hyperdiverse cichlid fishes" (Fages et al.)

**Metadata** contains important data and information for reproducing results of the manuscript. 

**Scripts** contains scripts for reproducing results of the manuscript. All R scripts have been executed in R v4.1.2.

### Mapping, QC and integration of single-cell data
Scripts 01, 02, 03, 04


### Clustering, cell type labeling and visualisation of integrated single-cell RNAseq data
Scripts 05, 06, 07, 08


### Mapping of bulk RNAseq data and deconvolution for characterisation anterior-posterior identity of epithelial cell populations  
Scripts 09, 10


### Transcriptome evolution of cell populations  
Scripts 11, 12, 13


### Associations between ecology and cell population abundances / gene expression profiles
Scripts 14, 15, 16

### Characterisation of ecologically relevant genes in enterocytes 1
Scripts 17, 18

**Necessary programs, databases and libraries to run scripts:**
Cell Ranger v5.0 (10X Genomics), 
Trimmomatic (v0.39), 
STAR (v2.7.3), 
HTSeq (v2.0), 
REVIGO web server (v1.8.1), 
EVEE-tools, 
STRING v11 database, 
BayesTraits (v3.0.2),
Python3 (v3.12) and oggmap Python package, 
R v4.1.2, 
R packages: SoupX (v1.6.2), ddqcR (v0.1), Seurat (v4.1.0), SciBet (v1.0), MetaNeighbor (v1.14), BayesPrism (v2.0), limma (v3.50.3), Go.db (v3.14), DESeq2 (v1.14), Augur (v1.0.3), caper (v1.0.3), speckle (v0.0.3), mvMORPH (v1.1.8), geiger (v2.0.7), coda (v.0.19.3), phytools (v2.1.1), ape (v5.7.1), ggplot2 (v3.5.1), scWGCNA, pheatmap (v1.0.12), dplyr (v1.1.4), reshape2 (v1.4.4), viridis (v0.6.5)
